src/Util.js

```diff
@@ -20,7 +20,7 @@ export function getBasemapStyleUrl (style, apikey) {
   return url;
 }
 
-export function getBasemapStyleV2Url (style, apikey, language) {
+export function getBasemapStyleV2Url (style, token, options) {
```

Comment:
I like how you have `style` and `token` are the required params, and then the optional items are in the `options` object. It might be good to call that out in a comment here, something like:

```suggestion
/**
 * Utility to establish a URL for the basemap styles API v2
 *
 * @param {string} style 
 * @param {string} token 
 * @param {Object} [options] Optional list of options: language, worldview, or places.
 * @returns {string} the URL
 */
export function getBasemapStyleV2Url (style, token, options) {
```

(we don't really require jsdoc style on this project but I don't think it hurts in this case)